bash_completion

```diff
@@ -2560,7 +2560,9 @@ __load_completion()
             compfile="$dir/$compfile"
             # Avoid trying to source dirs; https://bugzilla.redhat.com/903540
             if [[ -d $compfile ]]; then
-                echo "bash_completion: $compfile: is a directory" >&2
+                # Do not warn with . or .. (especially the former is common)
+                [[ $compfile == */.?(.) ]] ||
+                    echo "bash_completion: $compfile: is a directory" >&2
```

Comment:
This is just a comment so if you think it's unneeded, please merge it as is.

For example,

```suggestion
                if [[ $compfile != */.?(.) ]]; then
                    echo "bash_completion: $compfile: is a directory" >&2
                    echo "bash_completion: $dir: completionsdir should not contain subdirectories" >&2
                fi
```